### PR TITLE
Delete lateral tuning for TOYOTA

### PR DESCRIPTION
**Pull request by haraschax**

### CODE REVIEW

File: selfdrive/car/toyota/interface.py [hunk 1]
```diff
@@ -120,95 +116,109 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[], disabl
...
+      if candidate == CAR.AVALON:
+        torque_params = CarInterfaceBase.get_torque_params(CAR.AVALON_2019)
 
     elif candidate in (CAR.RAV4_TSS2, CAR.RAV4_TSS2_2022, CAR.RAV4H_TSS2, CAR.RAV4H_TSS2_2022):
       stop_and_go = True
       ret.wheelbase = 2.68986
       ret.steerRatio = 14.3
       tire_stiffness_factor = 0.7933
       ret.mass = 3585. * CV.LB_TO_KG + STD_CARGO_KG  # Average between ICE and Hybrid
-      set_lat_tune(ret.lateralTuning, LatTunes.PID_D)
```
Comment (krkeegan) [inline selfdrive/car/toyota/interface.py:131]: I think you are missing a default tune here.  For example, what tune would `CAR.RAV4H_TSS2` end up with?

Comment (krkeegan) [reply selfdrive/car/toyota/interface.py:131]: Apologies, I see that is what line 34 is doing.

File: selfdrive/car/toyota/interface.py [hunk 2]
```diff
@@ -120,95 +116,109 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[], disabl
...
 
       # 2019+ RAV4 TSS2 uses two different steering racks and specific tuning seems to be necessary.
       # See https://github.com/commaai/openpilot/pull/21429#issuecomment-873652891
+      # TODO different torque control for different firmware
```
[the remark this answers is not in the document]
Comment (HaraldSchafer) [reply selfdrive/car/toyota/interface.py:132]: Yeah I looked into it a little, but couldn't find any massive differences in the data. Will check with some people that have that eps

**Comment by krkeegan** [comment]
I have taken two short trips with this on a 2019 Rav4 Hybrid and it seems reasonable.  I previously had issues with the vehicle wandering out the lane at low speeds, I think the change from jerk to error for the friction control fixed this issue.

**Comment by HaraldSchafer** [comment]
@krkeegan Can you send me route, I am curious about which rav4 you have

**Comment by krkeegan** [comment]
Here you go, they are both short and don't have a lot of engagement:
2a96bb3fffd2e049|2022-06-15--07-11-20
2a96bb3fffd2e049|2022-06-12--11-20-02

File: selfdrive/car/toyota/interface.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (echo1001) [reply selfdrive/car/toyota/interface.py:132]: I have one of the odd EPSes and I gave the tune on this table a try. It felt like it was applying just a bit too much torque in some cases (like with the tighter curve I took for this screenshot). Other than that, it felt pretty good. I'm going to play with it this weekend to see what feels best and try to get you some data.

image

[the remark this answers is not in the document]
Comment (echo1001) [reply selfdrive/car/toyota/interface.py:132]: I finally had a windless night I could try this one. See route `7e8e854eb2c6127c|2022-06-18--01-18-30`

There was some minor oscillation and it felt like it jerked the steering wheel a bit too much.

`7e8e854eb2c6127c|2022-06-18--01-18-30--17` and `7e8e854eb2c6127c|2022-06-18--01-18-30--18` was a tighter curve. It initially applied too much torque and then corrected. 

Other than this, it felt fine, I would use it for most normal highway driving. Definitely a lot better with the change to use error.

**Comment by Taik** [comment]
@HaraldSchafer Here's a few drives from a 2020 RAV4 Hybrid TSS2:
```
4f07fcfe0eba45c4\|2022-06-15--21-27-24
4f07fcfe0eba45c4\|2022-06-15--17-19-27
4f07fcfe0eba45c4|2022-06-18--16-18-25
```

In general it definitely feels much more planted and can take some curves around my area nicely (see last drive). 

One of the drives were on a super windy day and I think it tried to compensate pretty aggressively, resulting in a pretty rough ride. Could that be from the high friction?

**Comment by HaraldSchafer** [comment]
Torqued will override this

**Pull request closed without merging**